pkg/volume/fc/fc_util.go

```diff
@@ -89,6 +89,45 @@ func findDisk(wwn, lun string, io ioHandler) (string, string) {
 	return "", ""
 }
 
+// given a wwid, find the device and associated devicemapper parent
+func findDiskWWIDs(b fcDiskMounter, wwid string, io ioHandler) (string, string) {
+	// Example wwid format:
+	//   3600508b400105e210000900000490000
+	//   <VENDOR NAME> <IDENTIFIER NUMBER>
+	// Example of symlink under by-id:
+	//   /dev/by-id/scsi-3600508b400105e210000900000490000
+	//   /dev/by-id/scsi-<VENDOR NAME>_<IDENTIFIER NUMBER>
+	// The wwid could contain white space and it will be replaced
+	// underscore when wwid is exposed under /dev/by-id.
+
+	fc_path := "scsi-" + strings.Replace(wwid, " ", "_", -1)
+	dev_id := "/dev/disk/by-id/"
+	if dirs, err := io.ReadDir(dev_id); err == nil {
+		for _, f := range dirs {
+			name := f.Name()
+			if strings.Contains(name, fc_path) {
+				disk, err := io.EvalSymlinks(dev_id + name)
+				if err != nil {
+					glog.V(2).Infof("fc: failed to find a corresponding disk from symlink[%s], error %v", dev_id+name, err)
+					return "", ""
```

Comment:
In my understanding, fc_path is unique under /dev/by-id.
Therefore, if symlink evaluation or checkDiskExist of scsi-3600508b400105e210000900000490000 fails, checking other links seems meaning less.

Also seems strings.Contains causes unexpected behavior because there is a possibility that choose partitioned device such as sda1, sda2, etc. I think we should check whether name matches fc_path completely.

ex.
```
lrwxrwxrwx 1 root root   9 Jul  7 19:47 scsi-3600508b400105e210000900000490000 -> ../../sdb
lrwxrwxrwx 1 root root   9 Jul  7 19:45 scsi-35000c500676fb68f -> ../../sda
lrwxrwxrwx 1 root root  10 Jul  7 19:45 scsi-35000c500676fb68f-part1 -> ../../sda1
lrwxrwxrwx 1 root root  10 Jul  7 19:45 scsi-35000c500676fb68f-part2 -> ../../sda2
lrwxrwxrwx 1 root root  10 Jul  7 19:45 scsi-35000c500676fb68f-part3 -> ../../sda3
lrwxrwxrwx 1 root root  10 Jul  7 19:45 scsi-35000c500676fb68f-part4 -> ../../sda4
lrwxrwxrwx 1 root root  10 Jul  7 19:45 scsi-35000c500676fb68f-part5 -> ../../sda5
lrwxrwxrwx 1 root root   9 Jul  7 19:45 wwn-0x5000c500676fb68f -> ../../sda
lrwxrwxrwx 1 root root  10 Jul  7 19:45 wwn-0x5000c500676fb68f-part1 -> ../../sda1
lrwxrwxrwx 1 root root  10 Jul  7 19:45 wwn-0x5000c500676fb68f-part2 -> ../../sda2
lrwxrwxrwx 1 root root  10 Jul  7 19:45 wwn-0x5000c500676fb68f-part3 -> ../../sda3
lrwxrwxrwx 1 root root  10 Jul  7 19:45 wwn-0x5000c500676fb68f-part4 -> ../../sda4
lrwxrwxrwx 1 root root  10 Jul  7 19:45 wwn-0x5000c500676fb68f-part5 -> ../../sda5
lrwxrwxrwx 1 root root   9 Mar 31 13:20 wwn-0x5001480000000000 -> ../../sr0
```

Any thought?